core/camel-management-api/src/main/java/org/apache/camel/api/management/mbean/ManagedConcurrentRequestsThrottlerMBean.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.api.management.mbean;
+
+import org.apache.camel.api.management.ManagedAttribute;
+
+public interface ManagedConcurrentRequestsThrottlerMBean extends ManagedProcessorMBean {
```

Comment:
Can we avoid having different mbeans - as there is only 1 EIP and ideally we should match that with the mbean too.

Keep the old mbean name, and then add attribute for mode, so you can see there which mode it uses.

Reply:
I am not sure, TBH. 

It seems that the bean works on top of the throttler itself. So, in some way, we would need to rework the code to make the throttler generic. 

I think we can do it, but I believe I might need to tackle the more critical issues first and then this.

Reply:
Done. Also implemented in a way that we could add other types of throttlers in the future.